examples/kubernetes/inventory/classes/component/mysql.yml

```diff
@@ -44,3 +48,5 @@ parameters:
 
         password_subvar: ?{gpg:targets/${target_name}/mysql/subvars@var1.password}
         password_sha256_subvar: ?{gpg:targets/${target_name}/mysql/subvars@var2.password_sha256}
+
+        password_vaultkv: ?{vaultkv:targets/${target_name}/mysql/vault_secret}
```

Comment:
One small observation about this. 
This will require all the VAULT related env vars to be set and a vault backend to be present in order for this line to be compiled. Which means that from a previous state of people being able to compile all examples, now they'll get an error in this one. 
I'd instead recommend you add this here in a comment just for reference.
And to keep your tests working, just introduce this line again in your tests setup() function (?). 
It would be best to keep the examples working for the majority of the users. :)